_source/_posts/2023-10-12-okta-documentation-chatbot.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: blog_post
+title: "How to Build an Okta Documentation Chatbot in Python"
+author: tanish-kumar
+by: internal-contributor
+communities: [python]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/okta-documentation-chatbot/oktanaut.jpeg
+type: 
+---
+
+In today's fast-paced world of technology, developer support is essential to ensure a seamless experience for users navigating intricate platforms and APIs. As a Developer Support Intern at Okta, I embarked on a challenging yet rewarding endeavor to create a solution that would streamline accessing and comprehending the Okta developer documentation. The result of my efforts is ✨Oktanaut✨, a versatile Python chatbot designed to assist users in harnessing the power of Okta's developer documentation, available at developer.okta.com. 
+
+{% img blog/okta-documentation-chatbot/oktadevforumsdk.jpeg alt:"oktadevforumsdk" height:"400" %}{: .center-image } 
+– Image source: <cite>[Okta Developer GitHub](https://raw.githubusercontent.com/oktadev/.github/main/images/okta-dev-header.png)</cite> 
+
+## Navigating Uncharted Waters
```

Comment:
Headers are sentence case

This can include keywords relatively easily i think.